### DISCUSSION TITLE
CC: Tweaked 1.113 for 1.20.1 and 1.21.1

### DISCUSSION THREAD

**Discussion by SquidDev** [cc-tweaked/CC-Tweaked#1947] [Announcements]
New block! For the first time in 7 years, CC:T has a new block! 

![A screenshot of a Minecraft lectern. It is facing away from the camera, so you cannot see the book on it.](https://github.com/user-attachments/assets/2a6713bd-4818-49ae-a65b-d64e26ded798)

No dear reader, I hear your confusion, but that's not a vanilla Minecraft lectern. It's a CC: Tweaked lectern. With printed pages on it!

![The same lectern, but from the other side. It is revealed that this actually has a spread of three pages on it.](https://github.com/user-attachments/assets/e27a6a8d-f26b-4906-8dca-b51e50515928)

**Comment by zyxkad** [comment]
What's the usage tho

**Reply by SquidDev** [reply]
It sparks joy.
